canonical/ubuntu.com
# Remove unsued python libraries and upgrade others

## Done

- Went through most packages and removed the ones that are not used anymore
- Upgraded the packages that I could from https://github.com/canonical/ubuntu.com/pull/12388

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - QA QA QA QA the site and make sure things still work!
